continuous_integration/docker/hadoop/files/etc/hadoop/conf.kerberos/core-site.xml

```diff
@@ -45,24 +45,4 @@
         <value>org.apache.hadoop.security.AuthenticationFilterInitializer</value>
     </property>
 
-    <property>
-        <name>hadoop.http.authentication.type</name>
```

Comment:
You disabled HTTP auth altogether? I agree with that.

Reply:
I saw it failing anyhow, so I figured removing it reduced complexity without downside.

Reply:
I'm confused. This is the default value anyhow so this didn't change much I think, but what does this config really do? I think it influences something that we may not be using?